addons/mail/static/src/components/rtc_configuration_menu/rtc_configuration_menu.js

```diff
@@ -14,13 +12,6 @@ export class RtcConfigurationMenu extends Component {
      */
     setup() {
         super.setup();
-        this.state = useState({
-            userDevices: undefined,
-        });
-    }
-
-    async willStart() {
-        this.state.userDevices = await browser.navigator.mediaDevices.enumerateDevices();
```

Comment:
I'm not sure why not filter here?
As this is a fix in stable, there's no reason to introduce `Rtc/userDevices`. Especially when it is introduced somewhere else in `16.0`+